### PR TITLE
[RN] Config object documentation

### PR DESCRIPTION
**Pull request by peterbrown0**

### CODE REVIEW

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
let's try creating all configuration examples as per this:

const countlyConfig = new CountlyConfig(SERVER_URL, APP_KEY);

//set the ... (describing the feature specific thing) 
countlyConfig.setDeviceId("TemporaryDeviceID");

await Countly.initWithConfig(countlyConfig);

File: react_native_bridged/next.md [hunk 1]
```diff
@@ -79,23 +82,29 @@ if(!await Countly.isInitialized()) {
     <a href="https://support.count.ly/hc/en-us/articles/900000908046-Getting-started-with-SDKs#how-to-validate-your-countly-integration" target="blank">here</a>.
   </p>
 </div>
+<h2>Init (Deprecated)</h2>
```
[the remark this answers is not in the document]
Comment (peterbrown0) [reply react_native_bridged/next.md]: Done

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Rework consent section, rework "init configuration" examples

**Review by turtledreams — APPROVED** [review]
looks cool. grammatically decent. just capitalized titles. header levels are correct. nothing that can staph a merge strike me eye

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
add documentation for the new attribution calls

**Pull request merged**